tests/test_basic.py

```diff
@@ -1289,6 +1289,20 @@ def test_jsonify_mimetype(app, req_ctx):
     assert rv.mimetype == "application/vnd.api+json"
 
 
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python3.7 or higher")
+def test_make_response_with_jsonified_dataclass(app, req_ctx):
+    from dataclasses import dataclass
+
+    @dataclass
+    class Data:
+        name: str
```

Comment:
that's a syntax error in older python versions, and thus breaking the whole file

Reply:
dataclasses can not be defined without those, any idea how to deal with that?

Reply:
Got it to work, did not know that it is possible to create dataclasses manually

Reply:
Thank you for the fast review.